src/components/datagrid/body/data_grid_body.tsx

```diff
@@ -271,6 +271,32 @@ InnerElement.displayName = 'EuiDataGridInnerElement';
 const INITIAL_ROW_HEIGHT = 34;
 const IS_JEST_ENVIRONMENT = global.hasOwnProperty('_isJest');
 
+/**
+ * getParentCellContent is called by the grid body's mutation observer,
+ * which exists to pick up DOM changes in cells and remove interactive elemnts
+ * from the page's tab index, as we want to move between cells via arrow keys
+ * instead of tabbing.
+ *
+ * So we start with a Node or HTMLElement returned by a mutation record
+ * and search its ancestors for a div[data-datagrid-cellcontent], if any
```

Comment:
small nit (I think? feel free to ignore if not valid)
```suggestion
 * and search its ancestors for a div[data-datagrid-cellcontent], if any,
```